lib/node_modules/@stdlib/blas/ext/base/dnansumors/lib/ndarray.js

```diff
@@ -42,27 +42,28 @@ var isnan = require( '@stdlib/math/base/assert/is-nan' );
 * var v = dnansumors( 5, x, 2, 1 );
 * // returns 5.0
 */
-function dnansumors( N, x, stride, offset ) {
+function dnansumors( N, x, strideX, offsetX ) {
 	var sum;
 	var ix;
 	var i;
 
+	ix = offsetX;
 	sum = 0.0;
 	if ( N <= 0 ) {
 		return sum;
 	}
```

Comment:
```suggestion
	sum = 0.0;
	if ( N <= 0 ) {
		return sum;
	}
	ix = offsetX;
```

Don't need to perform assignment until you actually need to.